tests/api/transactions/GET.test.js

```diff
@@ -6,13 +6,15 @@ const createUser = require("../../../helpers/Users/createUser");
 const createCourse = require("../../../helpers/Courses/createCourse");
 const createAppointment = require("../../../helpers/Appointments/createAppointment");
 const createTransaction = require("../../../helpers/Transactions/createTransaction");
+const searchQueryTrans = require("../../../helpers/Transactions/searchQueryTrans");
 
-let studentIdCookie, tutorIdCookie, foobarCookie;
+let studentIdCookie, tutorIdCookie, foobarCookie, wrongIdCookie;
```

Comment:
Why `wrongIdCookie`? This confused me for a bit. Something like `useNoTransaction` would be better.

Reply:
Resolved
https://github.com/The-Silent-Corner/sad-twt/pull/49/commits/a6e5a42d475d0f017602ac0b399cd9dfde4687c1#diff-24fa6350a0aad2d067849b46fd2b2f506df6ed12d0d10d23fca506ad58444e72R10